apps/mull-ui/src/app/components/chat-bubble/chat-bubble.tsx

```diff
@@ -1,26 +1,31 @@
 import React, { ReactChild } from 'react';
+import { User } from '../../../generated/graphql';
+import { avatarUrl } from '../../../utilities';
 import './chat-bubble.scss';
 
 export interface chatBubbleProps {
   isCurrentUser?: boolean;
   chatDate?: string;
-  userPicture?: string;
+  user: Partial<User>;
   children?: ReactChild;
   onClick?: React.MouseEventHandler<HTMLImageElement>;
 }
 
 export const ChatBubble = ({
   isCurrentUser = false,
   chatDate = '',
-  userPicture = '',
+  user,
   children = '',
   onClick = null,
 }: chatBubbleProps) => {
   return (
-    <div className="chat-container">
-      <p className="announcement-time">{chatDate}</p>
+    <div className="chat-bubble-container">
+      <div className="chat-bubble-header">
+        <p>{chatDate}</p>
+        <p className="chat-bubble-header-username">{user.name}</p>
```

Comment:
When looking at posts from a different user, the name should be on the left
![image](https://user-images.githubusercontent.com/33991121/113927017-ef557b00-97ba-11eb-93e8-db6b0a305c6f.png)

Reply:
Good catch, forgot about that